### PR TITLE
added flipkart hackathon,game development workshop and 3d printing wo…

### LINKED ISSUE #51: Create individual pages for each event.
Check the sample template and modify it as per your event page, just the basic look must be same.
Please mention here the name of event you are working on.

Refer: [Sample Template](https://kossiitkgp.github.io/events/detail.html)

### PR DESCRIPTION
**Pull request by ankita132**
partially fixes #51

### CODE REVIEW

File: events/game.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      <div class="row">
+        <div class="eight wide column">
+          <h1 class="ui header" style="font-size: 3.5rem">Event Overview</h1>
+          <p>In association with Technology Student's Gymkhana, Anurag Rana (2007 alumnus)
+             taught game development using Unity Game Enginer. He has been previously involved
+             in making the Indian Air Force app</p>
+
+        </div>
+      <div class="eight wide right floated column">
+         <h2 class="slide-center">Manual Slideshow</h2>
```
Comment (americast) [inline events/game.html:63]: I don't think this should be explicitly written.

File: events/flipkart.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+           from IIT Kharagpur and the present batch, regarding placements and career opportunities. KOSS is greatly indebted
+           to Flipkart and their team for their invaluable assistance. However, in retrospect it would not have been possible
+            to successfully pull off this event had it not been for the dedication and guidance of our beloved seniors
+            and mentors, Gaurav Jain and Smruti Ranjan Swain. All in all, it was a wonderful experience.</p>
+
+        </div>
+      <div class="eight wide right floated column">
+
+         <div class="slide-content slide-display-container">
+          <img class="mySlides" src="images/hackathon.jpg" style="width:100%;height:100%;">
```
Comment (americast) [inline events/flipkart.html:79]: We have just one image here. Could we do away with the slideshow ?

**Review by americast — CHANGES_REQUESTED** [review]
Can we have the slideshow automated ? Is there any specific reason you have kept them manual ?

**Comment by ankita132** [comment]
i will do that automated but the arrows should be present in the slideshow...UX is hampered when you dont have accessibility control

**Comment by americast** [comment]
@ankita132 Yeah, sure! Let the arrows remain, there's no harm.

**Pull request closed without merging**